build/build_deploy.sh

```diff
@@ -8,13 +8,20 @@ CURRENT_DIR=$(dirname "$0")
 
 BASE_IMG="managed-cluster-validating-webhooks"
 QUAY_IMAGE="quay.io/app-sre/${BASE_IMG}"
-IMG="${BASE_IMG}:latest"
-
+IMG="${BASE_IMG}:${VERSION_MAJOR}.${VERSION_MINOR}-${GIT_HASH}"
```

Comment:
Could we maybe have the same versioning scheme from the other osd operators?
MAJOR.MINOR.<commit count>-GIT_SHA